### PR TITLE
SQL-1659: add one-liner mentioning class name to readme

### PR DESCRIPTION
**Pull request by jchemburkar**
Additional documentation of the driver class name for clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
 
 ### Download and Install
 You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
-Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar).
+Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is "com.mongodb.jdbc.MongoDriver".
```
Comment (terakilobyte) [inline README.md:17]:
```suggestion
Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.
```

Comment (terakilobyte) [reply README.md:17]: Just a small suggestion, no need to accept.

**Review by terakilobyte — APPROVED** [review]
lgtm

**Comment by jchemburkar** [comment]
evergreen merge

**Pull request merged**